docker-compose.yml

```diff
@@ -24,9 +28,32 @@ services:
             - FM_DB=/srv/var/feed-master.bdb
             - FM_CONF=/srv/etc/fm.yml
             - TELEGRAM_TOKEN
+        # Uncomment the line below if you want to enable Telegram
+        # with file upload limit raised from default 50Mb to 2000Mb
+        #   - TELEGRAM_SERVER=http://telegram-bot-api:8081
         volumes:
             - ./var:/srv/var # mapped location to save status
             - ./_example/etc:/srv/etc # mapped location for config
             - ./_example/images:/srv/images # mapped location for images
         ports:
             - "8097:8080" # exposed on port 8097
+
+    telegram-bot-api:
```

Comment:
I would add a comment here describing this service is optional and needed for non-default TELEGRAM_SERVER only